test/navigation/NavigatorSpec.scala

```diff
@@ -17,96 +17,352 @@
+      "for the returns journey" - {
+
+        "for the ManufacturedPlasticPackagingPage" - {
+
+          "navigate to ManufacturedPlasticPackagingWeightPage when answer is Yes" in {
+            val answers = UserAnswers("id").set(ManufacturedPlasticPackagingPage, true)
+
+            navigator.nextPage(ManufacturedPlasticPackagingPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ManufacturedPlasticPackagingWeightController.onPageLoad(CheckMode)
 
-        case object UnknownPage extends Page
-        navigator.nextPage(UnknownPage,
-                           CheckMode,
-                           UserAnswers("id")
-        ) mustBe routes.CheckYourAnswersController.onPageLoad
+          }
+
+          "navigate to ImportedPlasticPackagingPage when answer is No" in {
+
+            val answers = UserAnswers("id").set(ManufacturedPlasticPackagingPage, false)
+
+            navigator.nextPage(ManufacturedPlasticPackagingPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ReturnsCheckYourAnswersController.onPageLoad
+
+          }
+        }
+
+        "for the ImportedPlasticPackagingPage" - {
+
+          "navigate to ImportedPlasticPackagingWeightPage when answer is Yes" in {
+            val answers = UserAnswers("id").set(ImportedPlasticPackagingPage, true)
+
+            navigator.nextPage(ImportedPlasticPackagingPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ImportedPlasticPackagingWeightController.onPageLoad(CheckMode)
+
+          }
+
+          "navigate to HumanMedicinesPlasticPackagingWeightPage when answer is No" in {
+
+            val answers = UserAnswers("id").set(ImportedPlasticPackagingPage, false)
+
+            navigator.nextPage(ImportedPlasticPackagingPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ReturnsCheckYourAnswersController.onPageLoad
+
+          }
+        }
+
+        "for the ManufacturedPlasticPackagingWeightPage" - {
+
+          "navigate to ImportedPlasticPackagingPage" in {
+            val answers = UserAnswers("id").set(ManufacturedPlasticPackagingWeightPage, 1000)
+
+            navigator.nextPage(ManufacturedPlasticPackagingWeightPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ReturnsCheckYourAnswersController.onPageLoad
+
+          }
+
+        }
+
+        "for the ImportedPlasticPackagingWeightPage" - {
+
+          "navigate to HumanMedicinesPlasticPackagingWeightPage" in {
+            val answers = UserAnswers("id").set(ImportedPlasticPackagingWeightPage, 1000)
+
+            navigator.nextPage(ImportedPlasticPackagingWeightPage,
+              CheckMode,
+              answers.get
+            ) mustBe routes.ReturnsCheckYourAnswersController.onPageLoad
+
+          }
+
+        }
+      }
 
+      "for the amend journey" - {
+        // TODO - implement me!!
```

Comment:
lol

Reply:
Indeed - something we can address after we get into staging :-)